pkg/plugins/generic/generic_plugin.go

```diff
@@ -233,6 +238,49 @@ func isCommandNotFound(err error) bool {
 	return false
 }
 
+// addToDesiredKernelArgs Should be called to queue a kernel arg to be added to the node.
+func (p *GenericPlugin) addToDesiredKernelArgs(karg string) {
+	if _, ok := p.DesiredKernelArgs[karg]; !ok {
+		glog.Infof("generic-plugin addToDesiredKernelArgs(): Adding %s to desired kernel arg", karg)
+		// element "uint" is a counter of number of attempts to set the kernel argument
+		p.DesiredKernelArgs[karg] = 0
+	}
+}
+
+// syncDesiredKernelArgs Should be called to set all the kernel arguments. Returns bool if node update is needed.
+func (p *GenericPlugin) syncDesiredKernelArgs() (bool, error) {
+	needReboot := false
+	if len(p.DesiredKernelArgs) != 0 {
```

Comment:
This looks like an optimization. All the following lines increase nesting. I prefer to have if len() == 0 { return false, nil} at the top since that's more in line with the rest of the repo.

Reply:
Good point, thanks! I made the changes.